test/functional/Valhalla/src_lw5/org/openj9/test/lworld/NullRestrictedTypeOptTests.java

```diff
@@ -52,7 +54,7 @@ public EscapeException(Object o) {
 	}
 
 	public static int result = 0;
-	public static PrimPair[] parr = new PrimPair[] { new PrimPair(3, 4), new PrimPair(0, 0), new PrimPair(0, 0) };
+	public static PrimPair![] parr = new PrimPair[] { new PrimPair(3, 4), new PrimPair(0, 0), new PrimPair(0, 0) };
```

Comment:
Thanks you're right. I've updated the three you pointed out.
`public static PrimPair![] parr = new PrimPair![] { new PrimPair(3, 4), new PrimPair(0, 0), new PrimPair(0, 0) };`
gave an assertion error in the latest lw5 compiler so I declared it as
`	public static FlattenablePair![] parr = new FlattenablePair![3];
	static {
		/* This syntax is a workaround since initializing the values of a NullRestricted array during 
		 * its declaration throws an assertion error in the lw5 compiler. */
		parr[0] = new FlattenablePair(3, 4);
		parr[1] = new FlattenablePair(0, 0);
		parr[2] = new FlattenablePair(0, 0);
	}`